src/video_transcoding/models.py

```diff
@@ -138,7 +176,7 @@ class Video(TimeStampedModel):
     duration = models.DurationField(blank=True, null=True)
 
     class Meta:
-        app_label = 'video_transcoding'
+        abstract = defaults.VIDEO_MODEL != 'video_transcoding.Video'
```

Comment:
It seems to be not transparent.May be define 2 models (VideoBase and Video) like others in the flle?

Reply:
But `Video.Meta.abstract` still will depend on `defaults.VIDEO_MODEL`. What's the point?

Reply:
class VideoBase:

Reply:
Pycharm and MyPy start complaining on strange errors. Let's leave it as is.